Makefile

```diff
@@ -21,32 +21,36 @@ deps_table_update:
 
 # Check that source code meets quality standards
 
-extra_quality_checks: deps_table_update
+extra_quality_checks:
 	python utils/check_copies.py
 	python utils/check_table.py
 	python utils/check_dummies.py
 	python utils/check_repo.py
-	python utils/style_doc.py src/transformers docs/source --max_len 119
-	python utils/class_mapping_update.py
 
 # this target runs checks on all files
 quality:
 	black --check $(check_dirs)
 	isort --check-only $(check_dirs)
+	python utils/custom_init_isort.py --check_only
 	flake8 $(check_dirs)
-	python utils/style_doc.py src/transformers docs/source --max_len 119 --check_only
 	${MAKE} extra_quality_checks
 
 # Format source code automatically and check is there are any problems left that need manual fixing
 
-style: deps_table_update
+extra_style_checks: deps_table_update
+	python utils/custom_init_isort.py
+	python utils/style_doc.py src/transformers docs/source --max_len 119
+	python utils/class_mapping_update.py
+
+# this target runs checks on all files
+style:
```

Comment:
this could just be:
```
style: extra_style_checks
```

for some reason it doesn't let me propose the edit across several lines

Reply:
unless you need that target to run after black and isort then it's perfect as it is.

Reply:
I am not 100% sure the scripts I wrote will work on badly formatted data, so if it's all the same to you, I would leave it at the end just to be sure it works.

Reply:
yes, yes, it's a perfectly good way